charts/dapr/README.md

```diff
@@ -46,3 +46,44 @@ To uninstall/delete the `dapr` release:
 ```
 helm uninstall dapr -n dapr-system
 ```
+
+## Configuration
+
+| Parameter                                 | Description                                                             | Default                 |
+|-------------------------------------------|-------------------------------------------------------------------------|-------------------------|
+| `global.registry`                         | Global Dapr docker image registry                                       | `docker.io/daprio`      |
+| `global.tag`                              | Global Dapr docker image version tag                                    | `0.10.0`                |
```

Comment:
default version is the version of the helm chart?

Reply:
it is runtime version. I changed to 0.11.0 for now.